libs/utils/executor.py

```diff
@@ -477,21 +477,21 @@ def _wload_init(self, tc, wl_idx):
         wload = self._wload_conf(wl_idx, wlspec)
 
         # Keep track of platform configuration
-        self.te.test_dir = '{}/{}:{}:{}'\
+        test_dir = '{}/{}:{}:{}'\
             .format(self.te.res_dir, wload.wtype, tc_idx, wl_idx)
-        os.system('mkdir -p ' + self.te.test_dir)
-        self.te.platform_dump(self.te.test_dir)
+        os.system('mkdir -p ' + test_dir)
```

Comment:
This is taken care of in a separate [commit](https://github.com/ARM-software/lisa/pull/202/commits/334055c952058598f8c2706f0a0bc15cc8a75dcb)